A43-sdk-authorization.md

```diff
@@ -322,35 +315,52 @@ different languages.
 
 #### C++
 
-1. Application enables SDK authorization in gRPC Server.
-
-- Policy is statically initialized
+1. C++ authorization provider API
```

Comment:
Instead of using numbering here, please use subsection headers.  For example:

```
##### C++ Authorization Provider API
```

```
##### C-core Authorization Provider API
```

```
##### Example: Using a Static Authz Provider in C++
```

And so on.

Same for all languages.

Reply:
Done.